src/TaskContext.js

```diff
@@ -46,8 +44,8 @@ export function TaskProvider({ children }) {
     }
     nextBoardId = allBoard[nextI];
 
-    const deleteone = newTasks[boardId].filter((task) => task.id !== id);
-    thisTasks[boardId] = [...deleteone]; // delete from current board
+    // delete from current board
+    thisTasks[boardId] = newTasks[boardId].filter((task) => task.id !== id);
   
     const moveTask = newTasks[boardId].filter((task) => task.id === id);
     thisTasks[nextBoardId].push(...moveTask) // add to des. board
```

Comment:
I think you are confused about the `...` operator
It's also called the spread operator. It's used to "spread" the values of an `Array` or `Object`

Use it when you want to create copies of something. In this instance it does not makes sense to use it.

Here `moveTask` is a single task, hence you could just say
```
thisTasks[next...].push(moveTask)
```

Reply:
when I remove `...` then I move task to other board its text doesn't show in a new board

Reply:
isn't `moveTask` of type `{ id, text}` ?

Reply:
moveTask type is an object of array, like `[{ id, text}]`